src/test/rustdoc/manual_impl.rs

```diff
@@ -65,7 +65,7 @@ impl T for S2 {
 }
 
 // @has manual_impl/struct.S3.html '//*[@class="trait"]' 'T'
-// @has  - '//details[@open=""]/div[@class="docblock"]' 'Docs associated with the S3 trait implementation.'
+// @has  - '//details/div[@class="docblock"]' 'Docs associated with the S3 trait implementation.'
```

Comment:
Can you also add another check where you ensure with `@!has` the same thing but with the `@open=""` attribute please?